src/scripts/main.js

```diff
@@ -1 +1,91 @@
 'use strict';
+
+let isClickedLeft = false;
+let isClickedRight = false;
+let timer;
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+function resetTimer() {
+  clearTimeout(timer);
+
+  timer = setTimeout(() => {
+    if (!isClickedLeft) {
+      firstPromise();
+    }
+  }, 3000);
+}
+
+document.addEventListener('mousedown', (e) => {
+  if (e.button === 0) {
+    isClickedLeft = true;
+    firstPromise();
+    resetTimer();
+  }
+
+  if (e.button === 2) {
+    isClickedRight = true;
+  }
+
+  if (isClickedLeft !== isClickedRight) {
```

Comment:
This condition will be true when either `isClickedLeft` or `isClickedRight` is true, but not both. This may lead to `secondPromise` being called when only one of the mouse buttons is clicked. If this is not the intended behavior, you might want to revise this condition.